app/Constants/MediawikiNamespaces.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace App\Constants;
+
+enum MediawikiNamespaces {
+    public const item = 120;
+    public const property = 122;
+    public const lexeme = 146;
+}
```

Comment:
Just out of curiosity: why are these living on an `enum` when they don't use `enum`  features? I.e. this could also be a `class` and nothing changes as far as I can tell. If it'd be an `enum` as shown in the docs, it would probably use backed cases:

```php
enum MediawikiNamespaces {
    case Item = 120;
    case Property = 122;
    case Lexeme = 146;
}
```

Reply:
Great question; I actually switched from an `interface` to an `enum` last minute after thinking I was being clever to use a shiny new 8.1 feature but then did it fairly incompetently.

Thanks for the suggestion. Note it was also necessary to explicitly call `->value` on the new `enum` in the `PageFetcher`